vrpn_3DConnexion.h

```diff
@@ -79,6 +79,18 @@ class VRPN_API vrpn_3DConnexion: public vrpn_Button_Filter, public vrpn_Analog {
 };
 #endif  // not VRPN_USE_HID
 
+class VRPN_API vrpn_3DConnexionWireless : public vrpn_3DConnexion {
+public:
+	vrpn_3DConnexionWireless(vrpn_HidAcceptor *filter, unsigned num_buttons,
```

Comment:
Prior code to mine reads
Line 27: #if defined(VRPN_USE_HID)
Line 27: ...
Line 30:   vrpn_3DConnexion(vrpn_HidAcceptor *filter, unsigned num_buttons,
Line 52: #else   // not VRPN_USE_HID
line 53: class VRPN_API vrpn_3DConnexion: public vrpn_Button_Filter, public vrpn_Analog {
line 54:   vrpn_3DConnexion(vrpn_HidAcceptor *filter, unsigned num_buttons,

vrpn_HidAcceptor  is used on line 30 when VRPN_USE_HID is declared and line 54 when VRPN_USE_HID is not declared.

I'll admit I've only been able to test this on Windows which has VPRN_USE_HID enabled. I'd rather test it everywhere but not sure I have enough knowledge for that right now.